c/parallel/src/nvrtc/nvjitlink_helper.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#define NVJITLINK_NO_INLINE
+#include <nvJitLink.h>
+#undef NVJITLINK_NO_INLINE
```

Comment:
This `#undef` surprises me slightly. I'd lean towards keeping it defined:

* Assuming `nvJitLink.h` is the only code from where this define is referenced: it does not matter.

* Assuming other Nvidia code also references this define: undefining here could lead to inconsistencies.